src/host.js

```diff
@@ -10,9 +10,12 @@ export default class Host extends Port {
 			throw new Error(`Unable to extract origin from "${src}"`);
 		}
 
-		var parent = document.getElementById(id);
-		if(parent === null) {
-			throw new Error(`Could not find parent node with id "${id}"`);
+		var parent = id;
+		if (typeof id === 'string') {
```

Comment:
From the consumer perspective, assuming they are writing ES6, it would look something like:
```
var host = new Host( () => document.getElementById('hostContainerId'), src, options );
// or
var host = new Host( () => hostContainerElement, src, options );
```
A little more for them to write, but I think it's worth it.  With ES5 I might consider adding helper functions (to limit the verbosity of the anonymous function) for these two cases, but with arrow functions they wouldn't add much value.